index.html

```diff
@@ -0,0 +1,178 @@
+      crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap"
+      rel="stylesheet" />
+
+    <link rel="stylesheet" href="dist/style.css" />
+    <title>Smart | Home</title>
+  </head>
+  <body>
+    <div class="container-all">
+      <!-- ================== Header ====================== -->
+
+      <header class="header-top flex flex-jc-b flex-center limit">
+        <p class="logo flex flex-center">
+          <img src="./app/assets/head.svg" alt="" />
+          <a href="index.html">Sm<span class="span">art</span></a>
+        </p>
+        <nav class="menu flex flex-al-c flex-jc-c">
+          <ul class="nav-links flex flex-jc-b">
+            <li><a class="nav-link" href="index.html">Home</a></li>
+            <li><a class="nav-link" href="about.html">About</a></li>
+            <li><a class="nav-link" href="#program">Program</a></li>
+            <li><a class="nav-link" href="#join">Join</a></li>
+            <li><a class="nav-link" href="#sponsor">Sponsor</a></li>
+            <li><a class="nav-link contact" href="#contact">Contact</a></li>
+          </ul>
+        </nav>
+
+        <div class="btn-container flex flex-center">
+          <span class="menu-btn"></span>
+        </div>
+      </header>
+
+      <main>
+        <!-- ================== Hero Section =================-->
+
+        <section class="hero flex flex-jc-c flex-col limit">
+          <div class="hero-text">
+            <header>
+              <p class="hero-top-desc">"Sharing our passion for learning with you."</p>
+              <h1 class="hero-title">Learn With Us</h1>
+            </header>
+            <p class="hero-desc">
+              We are more than a school <br />
+              We are a family.
+              <span>Next class starting next month</span>
+            </p>
+          </div>
+          <svg
+            class="arrow"
+            viewBox="0 0 16 132"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg">
+            <path
+              d="M7.29289 131.707C7.68341 132.098 8.31658 132.098 8.7071 131.707L15.0711 
+              125.343C15.4616 124.953 15.4616 124.319 15.0711 123.929C14.6805 
+              123.538 14.0474 123.538 13.6568 123.929L7.99999 129.586L2.34314 
+              123.929C1.95262 123.538 1.31945 123.538 0.928927 123.929C0.538402 
+              124.319 0.538402 124.953 0.928927 125.343L7.29289 131.707ZM7 -4.37114e-08L6.99999 
+              131L8.99999 131L9 4.37114e-08L7 -4.37114e-08Z"
+              fill="white"></path>
+          </svg>
+        </section>
+
+        <!-- ================ Main Program =================== -->
+
+        <section class="program limit flex flex-col flex-center" id="program">
+          <header class="program-header">
+            <h1>Main Program</h1>
```

Comment:
- [ ] Please note that it is SEO best practice to use only one `<h1>` tag for every page of your web app. The `<h1>` tag should contain the title of your page so that screen readers can easily identify what your page is about from the title. Please check [here](https://ahrefs.com/blog/h1-tag/#:~:text=tags%20improve%20accessibility-,H1%20tags%20helps%20search%20engines%20understand%20the%20page,what%20my%20page%20is%20about.%E2%80%9D) for more details.

Reply:
Thanks for this one, I was not aware.